hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieCreateHandle.java

```diff
@@ -235,10 +234,10 @@ protected void setupWriteStatus() throws IOException {
     stat.setNumInserts(insertRecordsWritten);
     stat.setPrevCommit(HoodieWriteStat.NULL_COMMIT);
     stat.setFileId(writeStatus.getFileId());
-    stat.setPath(new Path(config.getBasePath()), path);
+    stat.setPath(FSUtils.getLogicalRelativeFilePathStr(partitionPath, physicalPath.getName()));
```

Comment:
In `HoodieCommitMetadata` / `HoodieWriteStat`, Hudi would still store logical path there